src/safeds/data/tabular/containers/_column.py

```diff
@@ -546,6 +548,24 @@ def variance(self) -> float:
 
         return self._data.var()
 
+    # ------------------------------------------------------------------------------------------------------------------
+    # Transformations
+    # ------------------------------------------------------------------------------------------------------------------
+
+    def transform(self, transformer: Callable[[T], R]) -> Column[R]:
+        """
+        Apply a transform method to every data point.
+
+        Parameters
+        ----------
+        transformer : (...) -> Any.
+
+        Returns
+        -------
+        The transformed column.
+        """
+        return self._data.copy(deep=True).apply(transformer, convert_dtype=True)
```

Comment:
Is it necessary to copy here or does `apply` return a new `Series` anyway?

Also: This method is supposed to return a **Column** not a pandas Series.